test/sa_db_users.sql

```diff
@@ -42,6 +42,8 @@ GRANT SELECT,INSERT,UPDATE ON orders TO 'sa'@'localhost';
 GRANT SELECT,INSERT ON orderToAuthz TO 'sa'@'localhost';
 GRANT SELECT,INSERT ON requestedNames TO 'sa'@'localhost';
 GRANT SELECT,INSERT,DELETE ON orderFqdnSets TO 'sa'@'localhost';
+GRANT SELECT,INSERT,UPDATE ON authz2 TO 'sa'@'localhost';
+GRANT SELECT,INSERT ONN orderToAuthz2 TO 'sa'@'localhost';
```

Comment:
There's a small typo in this `GRANT`:
```suggestion
GRANT SELECT,INSERT ON orderToAuthz2 TO 'sa'@'localhost';
```

Reply:
+1